backend/src/data/models/course/course.model.ts

```diff
@@ -13,6 +16,19 @@ class Course extends Abstract {
 
   public 'courseCategoryId': number;
 
+  public static override get relationMappings(): RelationMappings {
+    return {
+      vendor: {
+        relation: Model.HasManyRelation,
```

Comment:
I think it's `HasOneRelation`

Reply:
I forgot to merge Farid's last changes, now it should be okay